test/IntegrationTests/CommonTests.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="CommonTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.IO;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using IntegrationTests.Helpers;
+using Xunit;
+
+namespace IntegrationTests;
+
+public class CommonTests
```

Comment:
How about renaming `BuildTests`?

Reply:
Sure. I do not have access to computer. I can fix it on Monday. If you want, fell free to change on my branch today.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1151/commits/a78ce3c39f4bdac02746fcd52172d684b045c4db